db.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-$env = parse_ini_file('.env');
+$env = parse_ini_file('.env.example');
```

Comment:
```suggestion
$env = parse_ini_file('.env');
```

你需要複製 `.env.example` 到 `.env`，編輯 DB parameters